### ISSUE TITLE
@padding not autocompleting for slicers

### ISSUE THREAD

**Issue by tremblap** [flucoma/flucoma-max#164]
probably due to not being in their documentation

**Comment by tremblap** [comment]
actually, the padding is not there at all - but it shows in the 'curves' - we need to think about that (although I am comfortable with no padding for the slicers)

**Comment by weefuzzy** [comment]
Yes, this was an explicit decision we all made when we added `padding` to the descriptors objects.

**Comment by tremblap** [comment]
I don't remember that decision but I am happy the parameter is there, and we should add a short entry in the documentation so it shows there too.

**Comment by weefuzzy** [comment]
Apparently in a sync meeting 30 March 2021, looking back over the PR in Bitbucket that introduced `@padding`

**Comment by tremblap** [comment]
ok - so I think we are saying the same thing:
- I remember padding is not there in slicers and this is the right decision
- I remember we thought of padding being important for all descriptor

re-reading trying to understand our 'disagreement' I think that my title is wrong. it is autocomplete for the new *features that is not there. I've edited the title - sorry for the confusion

**Comment by weefuzzy** [comment]
Ok, then this should transfer to flucoma-docs
